### DISCUSSION TITLE
How to investigate when I cannot access a specific website (domain) via openmptcprouter

### DISCUSSION THREAD

**Discussion by skmshd** [Ysurac/openmptcprouter#2292] [Q&A]
I started using openmptcprouter two weeks ago, so far it's great.

However, I recently noticed that I sometimes can not access websites (domains) via openmptcprouter. When this happens, I always get an "ERR_TIMED_OUT" message in my Chrome browser.
Are there any quick ways to investigate why this issue happens? It's obvious that Amazon or Netflix block access from VPS, but it is unlikely that a private blog site (e.g. [https://pcmanabu.com/](https://pcmanabu.com/)) would impose such restrictions.

Any advice is always welcome! Thanks.

**Comment by skmshd** [comment]
Just additional FYI: With OpenMTR, I can not access following sites with ERR_TIMED_OUT error.

- https://hldc.co.jp/blog/
- https://raspida.com/

Should I submit this as bug?

**Reply by Ysurac** [reply]
Check using ```curl -L https://pcmanabu.com/``` on the VPS that you have a long html result, same on router. This will to find where is the problem.
What is the proxy used ? Default is Shadowsocks.
You should also check Status->System log.

**Reply by skmshd** [reply]
Thanks for your advice, really appreciated, Ysurac.

```
root@vultr:~# curl -L https://pcmanabu.com/
curl: (28) Connection timed out after 300000 milliseconds
```

I am using Vultr and the command above was timeouted. Other sites I reported also have the same.
Any idea why this happens? Vultr is not a good VPS?
